app/src/main/java/org/openobservatory/ooniprobe/common/MKCollectorResubmitTask.java

```diff
@@ -34,15 +35,17 @@ public MKCollectorResubmitTask(A activity) {
 	}
 
 	private static void perform(Context c, Measurement m) throws IOException {
-		String input = FileUtils.readFileToString(Measurement.getEntryFile(c, m.id, m.test_name), StandardCharsets.UTF_8);
+		File file = Measurement.getEntryFile(c, m.id, m.test_name);
+		file.mkdirs();
```

Comment:
You should be creating the measurementDirectory not a directory with the name of the file.